### PR TITLE
Api documentation

### PR DESCRIPTION
**Pull request by patrickab**
This pull request introduces first a first impression about how the content of our API-Documentation could look like. However, a standard documentation scheme as mentioned in #38 should be applied. For REST API's, such as our flask server, [Swagger](https://swagger.io/resources/articles/documenting-apis-with-swagger/) might be a good solution.

### CODE REVIEW

File: prefq/templates/web_interface.html [hunk 1]
```diff
@@ -67,15 +67,18 @@
   </div>
 
     <script src="{{ url_for('static', filename='web_interface.js') }}"> </script>
+
```
Comment (timokau) [inline prefq/templates/web_interface.html:77]: This was already merged in #45. Could you rebase the PR on latest master?

**Review by timokau — CHANGES_REQUESTED** [review]
Looks good! Just one minor nitpick regarding the file extension change.

File: prefq/server.py [hunk 2]
```diff
@@ -59,14 +103,25 @@ def load_web_interface():
...
+    """
 
     print("\n\nServer: Starting receive_videos() [...]")
 
     print("Server: Receiving videos...")
     query_id = unquote(request.files.get("query_id").filename).strip('"')
-    left_filename = query_id + "-left.webm"
-    right_filename = query_id + "-right.webm"
+    left_filename = query_id + "-left.mpeg"
+    right_filename = query_id + "-right.mpeg"
```
[the remark this answers is not in the document]
Comment (patrickab) [reply prefq/server.py:124]: good point! this is just a relict related to #24

[the remark this answers is not in the document]
Comment (patrickab) [reply prefq/server.py]: i've just reverted these changes in a interactive rebase

**Review by timokau — APPROVED** [review]
Great, thank you!

**Pull request merged**